lib/enkf/enkf_main.c

```diff
@@ -691,10 +692,17 @@ static void deserialize_node( enkf_fs_type * fs,
                               const active_list_type * active_list,
                               matrix_type * A) {
   const enkf_config_node_type * config_node = ensemble_config_get_node( ensemble_config , key );
-  enkf_node_type * node = enkf_node_alloc( config_node );
+
   node_id_type node_id = {.report_step = target_step, .iens = iens  };
-  enkf_node_deserialize(node , fs , node_id , active_list , A , row_offset , column);
-  state_map_update_undefined(enkf_fs_get_state_map(fs) , iens , STATE_INITIALIZED);
+  enkf_node_type * node = enkf_node_alloc( config_node );
+
+  // If partly active, init node from source fs (deserialize will fill it only in part)
+  if(active_list_get_mode(active_list) == PARTLY_ACTIVE)
```

Comment:
I think this should be: `active_list_get_mode(active_list) != ALL_ACTIVE)` - it is not that difficult to envision the slightly pathological case where no elements are added to the vector and the mode is `INACTIVE` - which would fail now.

Reply:
ha, I had it like that before and decided to change it right before committing xD
I'll fix it